WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsViewModel.swift

```diff
@@ -275,8 +275,12 @@ final class WooShippingCreateLabelsViewModel: ObservableObject {
                 }
             }
 
-            group.addTask {
-                await self.loadShipmentsInfo()
+            let totalOrderItems = order.items.map(\.quantity).reduce(0, +)
+            if totalOrderItems > 1 {
+                // Only fetch shipments info if there are more than one order items.
+                group.addTask {
+                    await self.loadShipmentsInfo()
+                }
```

Comment:
Suggestion: We could move this `totalOrderItems` check logic into `loadShipmentsInfo` to keep the shipments-related logic inside one method.

Reply:
I plan to remove this check when working on #15309, so I'll keep this for now.